cmd/release-controller/sync.go

```diff
@@ -276,6 +276,15 @@ func countUnreadyReleases(release *Release, tags []*imagev1.TagReference) int {
 	return unreadyTagCount
 }
 
+func isReleaseFailing(tags []*imagev1.TagReference, maxUnready int) bool {
```

Comment:
move next to your dashboard then this change lgtm

Reply:
done